infra_tooling_account/infra_tooling_account/infra_tooling_common_stack.py

```diff
@@ -96,6 +116,21 @@ def create_timestream_db(self):
 
         return timestream_storage
 
+    def create_timestream_tables(self, timestream_storage):
+        retention_properties_property = timestream.CfnTable.RetentionPropertiesProperty(
+            magnetic_store_retention_period_in_days="365",
```

Comment:
Wouldn't be better to move these contants "365" and "240" into settings or contants.py file?

Reply:
Agree. Added trello task for future not to bloat current PR